Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/TwitterViewModelFactory.kt

```diff
@@ -0,0 +1,17 @@
+package com.mentalmachines.droidcon_boston.utils
+
+import android.content.Context
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.mentalmachines.droidcon_boston.data.Repository
+import com.mentalmachines.droidcon_boston.domain.TwitterUseCase
+import com.mentalmachines.droidcon_boston.views.social.TwitterViewModel
+
+class TwitterViewModelFactory(private val context: Context) : ViewModelProvider.NewInstanceFactory
+    () {
```

Comment:
Does AS allow this to be formatted this way?

Reply:
Yeah, my bad. Fixing this.